### PR TITLE
Use new auth + update README

### PR DESCRIPTION
**Pull request by windkomo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,17 +11,20 @@ npm install
 npm install -g ngrok
 ```
 
+ngrok is a is a command line tool that allows you to expose a localhost port to the internet through a secure tunnel. We use it for covenience here so you don’t have to host your bot on a server.
```
Comment (kevinadda) [inline README.md]: convinience

File: README.md [hunk 2]
```diff
@@ -11,17 +11,20 @@ npm install
...
+* The `APP_ID` and `APP_KEY` parameters comes from one of your apps on [Botfuel Developer Portal](https://app.botfuel.io/).
+
 * Run the following commands:  
 ```
 ngrok http 5000
-MESSENGER_VALIDATION_TOKEN=<...> MESSENGER_PAGE_ACCESS_TOKEN=<...> BOTMETER_USER_KEY=<...> node app.js
+MESSENGER_VALIDATION_TOKEN=<...> MESSENGER_PAGE_ACCESS_TOKEN=<...> APP_ID=<...> APP_KEY=<...> node app.js
 ```
-* You have to decide on the `MESSENGER_VALIDATION_TOKEN`'s value, and use it for the webhook token in Facebook for Developpers.  
+* You have to decide on the `MESSENGER_VALIDATION_TOKEN`'s value, and use it for the webhook token in Facebook for Developers.
```
Comment (kevinadda) [inline README.md]: `You have to decide on the `MESSENGER_VALIDATION_TOKEN`'s value` ? Why would you decide on a token?

File: app.js [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
-const botmeterLogger = require('@botfuel/botmeter-logger');
 
-const botmeterApiUrl = 'http://api.botmeter.io/';
+const messengerLogger = new BotmeterLogger({
+  appId: process.env.APP_ID,
+  appKey: process.env.APP_KEY,
+}).messenger;
 
-const botmeter = botmeterLogger(botmeterApiUrl, process.env.BOTMETER_USER_KEY).messenger;
+// This sample code is mostly taken from
```
Comment (kevinadda) [inline app.js]: `code sample`

File: app.js [hunk 4]
```diff
@@ -16,14 +20,17 @@ const callSendAPI = (senderId, requestBody) => {
...
   const response = {
     uri: 'https://graph.facebook.com/v2.6/me/messages',
     qs: { access_token: process.env.MESSENGER_PAGE_ACCESS_TOKEN },
     method: 'POST',
     json: responseJson,
   };
+
   request(response, () => {
-    botmeter.logDocument(requestBody, responseJson, (e, r) => {
+    // Function that logs to Botmeter
```
Comment (kevinadda) [inline app.js]: into Botmeter?

File: package.json [hunk 5]
```diff
@@ -10,6 +10,7 @@
   "dependencies": {
     "@botfuel/botmeter-logger": "latest",
```
Comment (kevinadda) [inline package.json]: remove this one ? Too early?

File: README.md (hunk 2 above)
Comment (windkomo) [reply README.md]: screen shot 2017-09-07 at 10 24 47

To create a page subscription, messengers bot developers have to enter a custom Verify Token field and then check it from their code 😃

File: app.js (hunk 3 above)
Comment (windkomo) [reply app.js]: Not sure about this, but I've made the change 😄

File: app.js (hunk 4 above)
Comment (windkomo) [reply app.js]: Not sure about this, but I've made the change 😄

File: README.md (hunk 2 above)
Comment (kevinadda) [reply README.md]: 😆

File: app.js (hunk 3 above)
Comment (kevinadda) [reply app.js]: Just checked, both are used,  my bad!

File: app.js (hunk 4 above)
Comment (kevinadda) [reply app.js]: 👍

**Pull request merged**